### PR TITLE
Update starship documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3491

### CODE REVIEW

File: docs-partials/starship-usage.mdx [hunk 1]
```diff
@@ -18,28 +18,32 @@ Some environment variables or Airflow Settings, like global environment variable
...
+8. Click **Pools**. In the table that appears, click **Migrate** for each connection that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
+9. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
+10. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
+11. Click **Environment variables**. In the table that appears, check the box for each environment variable that you want to migrate to Astro, then click **Migrate**. After the migration is complete, the status **Migrated ✅** appears.
+11. Click **DAG History**. In the table that appears, check the box for each environment variable that you want to migrate to Astro, then click **Migrate**. After the migration is complete, the status **Migrated ✅** appears.
```
Comment (lzdanski) [inline docs-partials/starship-usage.mdx:36]:
```suggestion
9. Click **Pools**. In the table that appears, click **Migrate** for each connection that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
10. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
11. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
12. Click **Environment variables**. In the table that appears, check the box for each environment variable that you want to migrate to Astro, then click **Migrate**. After the migration is complete, the status **Migrated ✅** appears.
13. Click **DAG History**. In the table that appears, check the box for each environment variable that you want to migrate to Astro, then click **Migrate**. After the migration is complete, the status **Migrated ✅** appears.
```

It's building even with the wrong numbering, but to make updates in future easier. I think this is right?

**Review by lzdanski — APPROVED** [review]
Not sure what's going on with the numbering which is correct in preview but wrong in source, but other than that LGTM

File: docs-partials/starship-usage.mdx [hunk 2]
```diff
@@ -18,28 +18,32 @@ Some environment variables or Airflow Settings, like global environment variable
...
-9. Click **Variables**.
-10. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
-11. Click **Environment variables**.
-12. In the table that appears, check the box for each environment variable that you want to migrate to Astro, then click **Migrate**. After the migration is complete, the status **Migrated ✅** appears.
+8. Click **Pools**. In the table that appears, click **Migrate** for each connection that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
+9. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
+10. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
```
Comment (fritz-astronomer) [inline docs-partials/starship-usage.mdx:34]:
```suggestion
9. Click **Variables**. In the table that appears, click **Migrate** for each variable that you want to migrate to Astro. After the migration is complete, the status **Migrated ✅** appears.
```
ty @astro-ben ! He gave a quick proof

**Comment by fritz-astronomer** [comment]
Mentioned in slack, but important change in the overall step ordering. 
The destination airflow needs to exist and have starship on it _prior_ to trying to use starship on the source airflow
https://astronomer.github.io/starship/#usage
@astro-ben fyi

File: docs-partials/directory-setup.mdx [hunk 3]
```diff
@@ -18,6 +18,14 @@
...
 
+4.  Add the following line to your Astro project `requirements.txt` file:
+
+    ```sh 
+    astronomer-starship
+    ```
+
+    When you deploy your code, this line installs the Starship migration tool on your Deployment so that you can migrate Airflow resources from your soure environment to Astro.
+
 4. (Optional) Run the following command to initialize a new git repository for your Astro project:
```
Comment (jwitz) [inline docs-partials/directory-setup.mdx:29]:
```suggestion
5. (Optional) Run the following command to initialize a new git repository for your Astro project:
```

**Pull request merged**